bin/assembleCoadd.py

```diff
@@ -20,7 +20,34 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
+from __future__ import print_function
 import lsst.pex.logging as pexLog
-from lsst.pipe.tasks.assembleCoadd import AssembleCoaddTask
+from lsst.pipe.tasks.assembleCoadd import AssembleCoaddTask, SafeClipAssembleCoaddTask
+import sys
+import argparse
 
-AssembleCoaddTask.parseAndRun()
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument('--legacyCoadd', action='store_true', default=False)
+parser.add_argument('-h','--help', action='store_true',default=False)
+result, extra = parser.parse_known_args(sys.argv)
+
+if result.help:
+    legacy_message = '''
+Meta Arguments:
+  --legacyCoadd         An option to run with the original assembleCoadd task which will
+                        not attempt to do any safe clipping.  If run, the config option
+                        'config.measurement.plugins["base_PixelFlags"].any = [] must be set
+'''
+    print(legacy_message)
+    SafeClipAssembleCoaddTask.parseAndRun(args=['--help']+extra[1:])
```

Comment:
Are you guaranteed to always in perpetuity get the same help message from `SafeClipAssembleCoaddTask` as from `AssembleCoaddTask`? If not, don't you have to check for the `legacyCoadd` argument and despatch the `--help` argument to the appropriate task?